.gitignore

```diff
@@ -0,0 +1,118 @@
+#
+# https://github.com/github/gitignore/blob/main/Node.gitignore
+#
+
+# TypeScript generated files
+dist/
+out/
+
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+vscode-crash-logs
+
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# test results
+test-results.xml
+xunit.xml
+xunit-unitTests.xml
+xunit-integrationTests.xml
+xunit-vscodeIntegrationTests.xml
+junit-aggregate
+junit-custom.xml
+junit-custom-unitTests.xml
+junit-custom-integrationTests.xml
+junit-custom-vscodeIntegrationTests.xml
+
+# nyc test coverage
+.nyc_output
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Bower dependency directory (https://bower.io/)
+bower_components
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directories
+node_modules/
+jspm_packages/
+
+# Optional npm cache directory
+.npm
+
+# Optional eslint cache
+.eslintcache
+
+# Optional REPL history
+.node_repl_history
+
+# Output of 'npm pack'
+*.tgz
+
+# Yarn Integrity file
+.yarn-integrity
+
+# dotenv environment variables file
+.env
+
+# Generated vscode package
+*.vsix
+
+# VS Code tests
+.vscode-test/
+
+# Eclipse
+.project
+
+# no keys
+*.key
+
+# SalesforceDX
+.sfdx/
+
+# Project test data
+
```

Comment:
@RitamAgrawal and @CristiCanizales when you check in your PRs, please add paths to test data here.